src/goto-programs/read_goto_binary.cpp

```diff
@@ -99,7 +99,7 @@ static bool read_goto_binary(
       for(unsigned i=0; i<elf_reader.number_of_sections; i++)
         if(elf_reader.section_name(i)=="goto-cc")
         {
-          in.seekg(elf_reader.section_offset(i));
+          in.seekg(std::streamoff{elf_reader.section_offset(i)});
```

Comment:
:question: Is this change meant to be here?

Reply:
Yes, because `section_offset` returns a `std::streampos` (which makes sense in other contexts).